health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/README.md

```
@@ -81,7 +85,8 @@
 ## Documentation
 
 Documentation Site (https://core.digit.org/guides/developer-guide/ui-developer-guide/digit-ui)
-Workbench Documentation(https://workbench.digit.org/platform/functional-specifications/workbench-ui)
+
+Campaign Module Documentation(https://docs.digit.org/health/0.3/setup/configuration)
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)